src/shared/components/lazyMobXTable/LazyMobXTable.tsx

```diff
@@ -954,10 +963,32 @@ export default class LazyMobXTable<T> extends React.Component<
                                 className="form-control tableSearchInput"
                                 style={{ width: 200 }}
                             />
+                            {this.props.showFilterClearButton &&
```

Comment:
maybe we we swap it with the magnifying glass instead of having two next to each other?

Reply:
good idea, updated